includes/class-wc-payments.php

```diff
@@ -1397,8 +1397,8 @@ public static function ajax_init_platform_checkout() {
 
 		$account_id = self::get_account_service()->get_stripe_account_id();
 
-		$platform_checkout_host = defined( 'PLATFORM_CHECKOUT_HOST' ) ? PLATFORM_CHECKOUT_HOST : 'https://pay.woo.com';
-		$url                    = $platform_checkout_host . '/wp-json/platform-checkout/v1/init';
+		$woopay_host = defined( 'PLATFORM_CHECKOUT_HOST' ) ? PLATFORM_CHECKOUT_HOST : 'https://pay.woo.com';
```

Comment:
`PLATFORM_CHECKOUT_HOST` seems to be defined only in WCPay, but might have dependencies on WooPay - do we plan to update this somewhere else?


UPDATE: I now see your comment [here](https://github.com/Automattic/woocommerce-payments/pull/6061/files#r1172509038). I'm assuming that we won't wait until the value will be changed on the WooPay side and update the constant value afterward as part of this PR, but instead accomplish this in another issue?

Reply:
@timur27 I think this another issue would be a better option to accomplish this. I think I mentioned this in previous comments, but we are trying to limit the scope of migration to file names, variables, functions, namespaces and other things that will be backward compatibly without changing config values, responses, return of functions and endpoints.